src/Core/Vault/Services/Implementations/CipherService.cs

```diff
@@ -139,8 +147,11 @@ public async Task SaveDetailsAsync(CipherDetails cipher, Guid savingUserId, Date
             else
             {
                 // Make sure the user can save new ciphers to their personal vault
```

Comment:
I know it's not part of your scope, but can we replace this comment with a meaningful variable name? I think your variable `IsPersonalVaultRestricted` is a good option.